blockchain/types/block_test.go

```diff
@@ -445,6 +445,15 @@ func TestEIP1559BlockEncoding(t *testing.T) {
 	}
 }
 
+func TestHeaderSizeCalc(t *testing.T) {
+	h := genMagmaHeader()
+	originSize := h.Size()
```

Comment:
What about using pre-calculated constant value instead of `h.Size()`?
```
	h := genMagmaHeader()
	originSize := `sizeOfTestHeader`
	assert.Equal(t, originSize, h.Size())

	diff := common.StorageSize(len(h.Governance) + len(h.Extra) + len(h.Vote) + h.BaseFee.BitLen())
	h.BaseFee, h.Governance, h.Extra, h.Vote = nil, nil, nil, nil
	assert.Equal(t, originSize, h.Size()+diff)
```

Reply:
I have suggested about using pre-calculated value instead of `h.Size()`
`originSize := h.Size()` -> `originSize := 500` (500 is just an example)

Reply:
Could you please explain the specific aspects or factors that lead you to manually perform calculations instead of using the existing function?

Reply:
I'm not sure what this test is trying to verify. `originSize` is `h.Size()`, and it should be equivalent to `h.Size()+diff`. Does it mean that we're testing if `diff` is 0?

I agree with yumiel's suggestion. The purpose of this test is supposed to test if `h.Size()` outputs the right answer (i.e., 정답).
And the size of genMagmaHeader() has one right answer, say, 500. That is where the constant comes from.